### PR TITLE
COMMERCE-7697, COMMERCE-7707, COMMERCE-7705, COMMERCE-7706

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/COMMERCE-7697
https://issues.liferay.com/browse/COMMERCE-7707
https://issues.liferay.com/browse/COMMERCE-7705
https://issues.liferay.com/browse/COMMERCE-7706

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase [hunk 1]
```diff
@@ -60,4 +60,86 @@ definition {
 			ImportExport.assertDownloadedFileName(downloadedFile = "csv_template.csv");
 		}
     }
+
+	@description = "Verify user can not upload non CSV files"
```
Comment (nikki-pru) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase]: @AfonsoHSantos , please check the file names uploaded in this test. Example, `Document_1.pdf` etc. In another commit, the files added are `Commerce.pdf`, etc

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
Testing here: https://github.com/nikki-pru/liferay-portal/pull/287

**Pull request closed without merging**